docs/source/multi_execution_experiments.rst

```diff
@@ -37,6 +46,29 @@ It also possible to use a combination of the range and choices operators to setu
 
    experiment_definition = experiment_definition.with_modifiers({'PI.k': Choices(10, 20),'PI.Ti': Range(10, 100, 3)})
 
+Expansion types
++++++++++++++++
+
+The operators could be further customized with expansion algorithms. Expansion methods define 
+how cases of an experiment are selected based on the respective operators. The following expansion
+algorithms are supported:
+
+1. :class:`Full-factorial<.modelon.impact.client.experiment_definition.expansion.FullFactorial>`: Creates 
+an experiment with all possible combinations of the respective operators. Supported operators are Range and Choices.
```

Comment:
Add: Full-factorial is the default and does not need to be specified explicitly.